### PR TITLE
Add support for null values

### PR DESCRIPTION
**Pull request by tonyraoul**

### CODE REVIEW

File: packages/typeorm-class-validator-is-uniq/__tests__/index.ts [hunk 1]
```diff
@@ -88,6 +90,29 @@ Array [
...
+        repo.create({
+          email: 'doe@example.com',
+          department: null,
+          company: 'ACME',
+        }),
+      ];
+      await repo.save(entities);
+    });
+    it('never consider null as unique value ', async () => {
+      const entity = await repo.findOneOrFail({ email });
```
Comment (mayadafahim) [inline packages/typeorm-class-validator-is-uniq/__tests__/index.ts]: email here is not null

Comment (tonyraoul) [reply packages/typeorm-class-validator-is-uniq/__tests__/index.ts]: email is not null but we need to check for validated values, it's a redundant query to match other tests.

**Comment by kirpichenko** [comment]
Maybe it makes sense to make this check optional? Smth like @IsUniq({ skipNulls: true }). WDYT?

**Comment by tonyraoul** [comment]
@kirpichenko doesn't this package already integrates with typeorm only ? that's its default behavior.
also 
```
  findOne({ where: null })
```
is not a defined behaviour iirc, so in-case of nulls should we keep it as it is ?

**Comment by kirpichenko** [comment]
yes. it's typeorm only. That was actually a question should we keep it or not? I would probably keep bc who knows how it's used on other services. Maybe somewhere it's not allowed to to have two null fields

**Pull request merged**